script/main.py

```diff
@@ -4,15 +4,20 @@
 
 
 def fetch_argument():
-  parser = argparse.ArgumentParser()
+  parser = argparse.ArgumentParser()   
+  parser.add_argument('--driver', help='entry driver')
   parser.add_argument("url", help="entry page url")
-  args = parser.parse_args()
-  return(args.url)
+  args = parser.parse_args()   
+  if args.driver  == 'chrome':
```

Comment:
It is better if function do only 1 thing. For example this function is responsible for parsing command line arguments.
It shouldn't also create driver. 

Create separate function `create_driver(name)` which will create driver based on parameter and call it from `main`